benchmark/test_transformer_engine_perf.py

```diff
@@ -8,6 +8,8 @@
 try:
     from transformer_engine.pytorch import cpp_extensions as tex
 
+    # note: the transformer_engine has its' own logging which may have conflict with flag_gems logging
```

Comment:
so ... what do you mean by conflict?
both of them are logging to stdout/stderr?

what's your intention by adding this note here?

Reply:
Great. Thanks for the elaborated feedback. We'll track this issue.